workloads/clearcontainers-3.x.yaml

```diff
@@ -182,44 +96,14 @@ runcmd:
  - {{template "ENV" .}}apt-get install qemu-system-x86 -y
  - {{endTaskCheck .}}
 
- - {{beginTask . "Installing xorriso"}}
- - {{template "ENV" .}}apt-get install xorriso -y
- - {{endTaskCheck .}}
-
  - {{beginTask . "Installing Clear Containers development tools"}}
  - {{template "ENV" .}}apt-get install build-essential python zlib1g-dev libcap-ng-dev libglib2.0-dev libpixman-1-dev libattr1-dev libcap-dev autoconf libtool libjson-glib-dev uuid-dev check bats libdevmapper-dev file apt-utils wget valgrind lcov libmnl-dev cppcheck libtap-formatter-html-perl -y
  - {{endTaskCheck .}}
 
- - {{beginTask . "Installing CRI-O dependencies"}}
- - {{template "ENV" .}}apt-get install -y socat libgpgme11 libostree-1-1
- - {{endTaskCheck .}}
-
- - {{beginTask . "Installing K8S"}}
- - {{template "ENV" .}}apt-get install -y kubelet=1.7.5-00 kubeadm=1.7.5-00 kubectl=1.7.5-00
- - {{template "ENV" .}}sudo apt-mark hold kubelet kubeadm kubectl
- - {{endTaskCheck .}}
-
- - {{beginTask . "Installing CRI-O"}}
- - {{template "ENV" .}}apt-get install -y cri-o
- - {{endTaskCheck .}}
-
  - {{beginTask . "Auto removing unused development components"}}
  - {{template "ENV" .}}apt-get auto-remove -y
  - {{endTaskCheck .}}
 
- - {{beginTask . "Configuring CRIO for Clear Containers"}}
- - sudo sed -i 's/runtime_untrusted_workload = ""/runtime_untrusted_workload = "\/usr\/bin\/cc-runtime"/' "/etc/crio/crio.conf"
- - {{endTaskCheck .}}
-
- - {{beginTask . "Configuring K8S for CRIO"}}
- - sudo sed -i '/KUBELET_CADVISOR_ARGS/a Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=/var/run/crio.sock --runtime-request-timeout=30m"' "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
- - {{endTaskCheck .}}
-
- - {{beginTask . "Start CRI-O daemon"}}
- - sudo systemctl enable crio
- - sudo systemctl start crio
- - {{endTaskCheck .}}
-
  - {{beginTask . "Installing Go development utils"}}
  - sudo -u {{.User}} {{template "ENV" .}} GOPATH={{template "GOPATH" .}} /usr/local/go/bin/go get github.com/mattn/goveralls golang.org/x/tools/cmd/cover github.com/pierrre/gotestcover github.com/fzipp/gocyclo github.com/gordonklaus/ineffassign github.com/golang/lint/golint github.com/client9/misspell/cmd/misspell github.com/ciao-project/ciao/test-cases github.com/opencontainers/runc/libcontainer/configs
```

Comment:
Which of these, if any are necessary for CC? They look like CIAO development tools.

Reply:
Apart from test-cases these are generic go development tools which are handy to have locally.  However, it might be better to replace all of these with gometalinter.